### ISSUE TITLE
Handle secondary options with only severity level

### ISSUE THREAD

**Issue by paulpjryan** [kristerkari/stylelint-z-index-value-constraint#13]
Providing rules: 
```
"rules": {
  "plugin/z-index-value-constraint": [{ max: 100 }, {severity: "error" }]
}
```
causes a `TypeError`. This is likely related to this check: https://github.com/kristerkari/stylelint-z-index-value-constraint/blob/master/index.js#L60

I have this working by providing these options:
```
"rules": {
  "plugin/z-index-value-constraint": [{ max: 100 }, { ignoreValues: [-1], severity: "error" }]
}
```
but it would be ideal to be able to just provide a severity level.

**Comment by kristerkari** [comment]
Thanks @paulpjryan! It's true that I haven't considered that the serverity level could be passed. 

I'll create a fix for it.

**Comment by kristerkari** [comment]
Fixed in v1.1.0
https://github.com/kristerkari/stylelint-z-index-value-constraint/releases/tag/1.1.0
